src/openms/include/OpenMS/TRANSFORMATIONS/FEATUREFINDER/MultiplexDeltaMasses.h

```diff
@@ -57,16 +57,48 @@ namespace OpenMS
   class OPENMS_DLLAPI MultiplexDeltaMasses
   {
     public:
+    
+    /**
+     * @brief set of labels associated with a mass shift
+     * 
+     * For example, a set of SILAC labels [Lys8, Lys8, Arg10] would
+     * result in a +26 Da mass shift.
+     */
+    typedef std::multiset<String> LabelSet;
+
+    /**
+     * @brief mass shift with corresponding label set
+     */
+    struct DeltaMass {
```

Comment:
@timosachsenberg How about `getDeltaMasses()[i].delta_mass` to get a particular delta mass?